relay/relay-server.md

```diff
@@ -10,9 +10,9 @@ The following definitions are shared concepts across all JSON-RPC methods for th
 
 - **topic** - a target topic for the message to be subscribed by the receiver.
 - **message** - a plaintext message to be relayed to any subscribers on the topic.
-- **ttl** - a storage duration for the message to be cached server-side in **seconds** (aka time-to-live). **(0 - no caching, -1 - forever)**
-- **tag** - a label that identifies which api sent a message. It allows collecting metrics. **(0 - unknown, 1 - sign, 2 - chat, 3 - auth, 4 - push)**
-- **prompt** - a flag that identifies whether the server should trigger a notification webhook to a client through a push server **(true/false)**
+- **ttl** - a storage duration for the message to be cached server-side in **seconds** (aka time-to-live).
```

Comment:
As discussed i think deriving the TTL form the tag is the better option rather than letting the client define this on a per message basis. As a user of the walletconnect protocol, this extra cognitive load confuses me. I'd rather have it implicit and only worry about the tags.

This also opens up more DDOS avenues.

It's ultimately not in line with our principle of being developer friendly.